Tests/js/views/ez-subitemlistview.html

```diff
@@ -33,7 +67,7 @@
         filter: loaderFilter,
         modules: {
             "ez-subitemlistview": {
-                requires: ['ez-templatebasedview', 'ez-asynchronousview', 'event-tap'],
+                requires: ['ez-templatebasedview', 'ez-asynchronousview', 'event-tap', 'event-valuechange'],
```

Comment:
array-extras should also be added to the list (for `Array.find`), also the same changes should be applied to yui.yml.